wd_dh.c

```diff
@@ -224,6 +225,13 @@ int wd_dh_init2_(char *alg, __u32 sched_type, int task_type, struct wd_ctx_param
 		goto out_clear_init;
 	}
 
+	dh_ctx_params.bmp = inner_bmp = numa_allocate_nodemask();
```

Comment:
Try not to use this continuous use of equal signs assignment

Reply:
fix now